### PR TITLE
DOCS-9547-PMDocReveiw-x12Settings

### PR DESCRIPTION
**Pull request by emileemooreVT**
Prepping doc for editorial review

### CODE REVIEW

File: modules/ROOT/pages/configure-partner.adoc [hunk 1]
```diff
@@ -4,11 +4,11 @@ The Partner Manager *Partners* page displays configured trading partner profiles
...
 
-* <<create-partner,Create and configure new trading partners>>
+* <<create-partner,Create and configure new partners>>
 * <<view-partner-details,View the details>> of a partner
-* xref:partner-manager-identifiers.adoc[Add the EDI and AS2 identifiers] used by your trading partner.
-* Add digital xref:Certificates.adoc[Certificates] associated with your trading partner.
-* xref:create-endpoint.adoc[Create] and manage source and target endpoints owned by the trading partner.
+* xref:partner-manager-identifiers.adoc[Add the EDI and AS2 identifiers] used by your partner.
+* Add digital xref:Certificates.adoc[certificates] associated with your partner.
+* xref:create-endpoint.adoc[Create] and manage source and target endpoints owned by the partner.
```
Comment (priyaayyar) [inline modules/ROOT/pages/configure-partner.adoc:11]: owned by your partner?

File: modules/ROOT/pages/configure-partner.adoc [hunk 2]
```diff
@@ -21,18 +21,22 @@ To create a new partner, you need the following information:
 [[create-partner]]
 == Create a Partner
 
+=== Prerequisite
```
Comment (priyaayyar) [inline modules/ROOT/pages/configure-partner.adoc:24]: I know that there is only one prereq, but I would still keep the heading as Prerequisites.

File: modules/ROOT/pages/configure-partner.adoc [hunk 3]
```diff
@@ -21,18 +21,22 @@ To create a new partner, you need the following information:
...
 Before you create your first trading partner, ensure you configure your xref:setup-payload-storage-API.adoc[payload storage API].
```
Comment (priyaayyar) [inline modules/ROOT/pages/configure-partner.adoc:25]: ensure that you configure..

File: modules/ROOT/pages/configure-partner.adoc [hunk 4]
```diff
@@ -21,18 +21,22 @@ To create a new partner, you need the following information:
...
 
-. From within your Anypoint Partner Manager environment, select *Partners* from the left navigation.
-. Click *New partner*.
-. In *Name*, enter a name that easily identifies your partner.
-. Select an identifier from the *Type* list:
+=== Steps
+. In Partner Manager, select *Partners* in the left navigation.
```
Comment (priyaayyar) [inline modules/ROOT/pages/configure-partner.adoc:28]: From the left navigation menu in Partner Manager, select *Partners* > *New partner*?

File: modules/ROOT/pages/configure-partner.adoc [hunk 5]
```diff
@@ -21,18 +21,22 @@ To create a new partner, you need the following information:
...
+.. In the *Name* field, enter a name that easily identifies your partner.
+.. In the *Type* list, select one of the following identifiers:
 * *AS2 Identity*
 * *Reference ID*
 * *X12-GS*
 * *X12-ISA*
-. In *Identifier Value*, enter a value to uniquely identify your partner.
-. If you selected *X12-ISA* for *Type*, you must also select a *Qualifier* value from the list.
+.. In the *Identifier Value* field, enter a value to uniquely identify your partner.
+.. Optionally, if you selected *X12-ISA* for *Type*, you must also select a *Qualifier* value from the list.
```
Comment (priyaayyar) [inline modules/ROOT/pages/configure-partner.adoc]:
```suggestion
.. Optionally, if you selected *X12-ISA* for *Type*, also select a *Qualifier* value from the list.
```

File: modules/ROOT/pages/configure-partner.adoc [hunk 6]
```diff
@@ -43,17 +47,18 @@ You can now xref:message-flows.adoc[set up message flows] for this trading partn
 . In Partner Manager, select *Partners* in the left navigation menu.
 . Click the partner whose profile you want to update.
 . Under the *About* section, click *Edit* to add or update the following information:
-* *Description* +
+* *Identity*
+** *Description* +
 Provide a brief description of your partner's business, what kind of trading relationship your organization has with them, and so on.
-* *Website* +
+** *Website* +
 Enter a valid URL to your trading partner's website.
```
Comment (priyaayyar) [inline modules/ROOT/pages/configure-partner.adoc:54]: Is it URL to or URL for?

File: modules/ROOT/pages/x12-identity-settings.adoc [hunk 7]
```diff
@@ -32,78 +30,74 @@ Valid values::
...
 
 == Delete Identifiers
 
 You might want to delete an identifier when it is no longer needed.
-Consider that deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner.
-Ask yourself the following when deleting an identifier: Why do you want to delete it? Do you need to replace it with another identifier?
-Remember that there must always be at least one identifier per message flow.
-
-After successfully deleting an identifier, you can create a new identifier with the same value as the one you deleted.
+Deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner. Remember that there must always be at least one identifier per message flow.
```
Comment (priyaayyar) [inline modules/ROOT/pages/x12-identity-settings.adoc:90]: Remember that there must..passive sentence?

You must always configure one identifier per message flow?

File: modules/ROOT/pages/x12-identity-settings.adoc [hunk 8]
```diff
@@ -32,78 +30,74 @@ Valid values::
...
 
+Ask yourself the following when deleting an identifier: Why do you want to delete it? Do you need to replace it with another identifier?
```
Comment (priyaayyar) [inline modules/ROOT/pages/x12-identity-settings.adoc:92]: Before you delete an identifier, ensure that you clearly know why you are deleting the identifier and whether you need to replace it with another identifier?

File: modules/ROOT/pages/x12-identity-settings.adoc [hunk 9]
```diff
@@ -32,78 +30,74 @@ Valid values::
...
 
-. If you need to replace the identifier, record the values and qualifier type needed.
-
-. Choose the partner name and click *Receive from <Partner name>* to expand it.
+After successfully deleting an identifier, you can create a new identifier with the same value as the one you deleted. If you need to replace the identifier you are deleting, record the values and qualifier type needed.
```
Comment (priyaayyar) [inline modules/ROOT/pages/x12-identity-settings.adoc:94]:
```suggestion 
If you need to replace the identifier you are deleting, record the values and qualifier type that you need?
The second part of the sentence seems odd.
```

File: modules/ROOT/pages/x12-identity-settings.adoc [hunk 10]
```diff
@@ -32,78 +30,74 @@ Valid values::
...
 
-. Select an identifier to the message flow, and click *See All*.
+. In the left navigation, select *Partners*.
```
Comment (priyaayyar) [inline modules/ROOT/pages/x12-identity-settings.adoc:96]: If you kept "menu" in the previous suggestion, replicate here.

File: modules/ROOT/pages/x12-receive-read-settings.adoc [hunk 11]
```diff
@@ -4,7 +4,7 @@ Partner Manager enables you to configure the X12 acknowledgments, validation rul
 
 == Configure the Receive Settings
 
-. In the left navigation menu, select the partner for whom to configure the send settings.
+. In the left navigation menu, select the partner for whom you want to configure the settings.
```
Comment (priyaayyar) [inline modules/ROOT/pages/x12-receive-read-settings.adoc:7]: You specify "menu" here, which seems correct. When you use "the" it applies to a noun/object. So I would think we need the word menu.

File: modules/ROOT/pages/configure-partner.adoc (hunk 4 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/configure-partner.adoc:28]: A new page appears when you select **Partners**, so I think I will leave this construction as is.

File: modules/ROOT/pages/x12-receive-read-settings.adoc (hunk 11 above)
Comment (emileemooreVT) [reply modules/ROOT/pages/x12-receive-read-settings.adoc:7]: This is one of the sections that will likely be rewritten - I am going to leave this here for now.

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**